sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -390,6 +416,18 @@ clang++ -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice \
 This `simple-sycl-app.exe` application doesn't specify SYCL device for
```

Comment:
This `simple-sycl-app.exe` application doesn't specify a SYCL device for execution, so the SYCL runtime will use `default_selector` logic to select one of the accelerators available in the system or the SYCL host device.

Do we need to say what the SYCL host device is. People are unlikely to know.

Reply:
Added a better explanation in 7d027efeda7845a8d6df6f7fa109d4c73451ea61